bitnami/kafka/README.md

```diff
@@ -600,6 +609,9 @@ docker run --name kafka-3 \
   -e KAFKA_CFG_LISTENERS=PLAINTEXT://:9092,CONTROLLER://:9093 \
   -e KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP=CONTROLLER:PLAINTEXT,PLAINTEXT:PLAINTEXT \
   -e KAFKA_CFG_CONTROLLER_QUORUM_VOTERS=0@kafka-0:9093,1@kafka-1:9093,2@kafka-2:9093 \
+  -e KAFKA_CFG_OFFSETS_TOPIC_REPLICATION_FACTOR=3 \
+  -e KAFKA_CFG_TRANSACTION_STATE_LOG_REPLICATION_FACTOR=3 \
+  -e KAFKA_CFG_TRANSACTION_STATE_LOG_MIN_ISR=2 \
```

Comment:
Personally, I feel this whole section for the three run commands should be replaced with `compose up`

Reply:
I do agree as well, but as we have both this and the `docker-compose/docker-compose-cluster.yml` files, it's up to the user to choose their desired path :)